Application.hs

```diff
@@ -84,13 +82,13 @@ version = $(mkVersion)
 mkYesodDispatch "App" resourcesApp
 
 -- probably not thread safe
-withEnv :: (MonadIO m) => ByteString -> ByteString -> m a -> m a
+withEnv :: (MonadIO m) => String -> String -> m a -> m a
 withEnv k v action = do
-    original <- liftIO $ getEnv k
+    original <- liftIO $ lookupEnv k
```

Comment:
Had to change from getEnv (throws error if not found) to lookupEnv (returns a Maybe).